### PR TITLE
added intregration tests for AddPeer on Monitor API

### PR DESCRIPTION
**Pull request by engelmi**
Relates to: https://github.com/eclipse-bluechi/bluechi/issues/187

Added integration tests to validate that the implemented API for adding and removing peers on monitors works as expected.

*Note:*
I am not 100% happy about the current solution for starting two processes (which are necessary to get different bus IDs), but couldn't come up with a more elegant solution - all other options tend to be even more complex.

### CODE REVIEW

File: src/bindings/python/setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def readme():
 
 setup(
     name="bluechi",
-    version="0.6.0",
+    version="0.7.0",
```
Comment (dougsland) [inline src/bindings/python/setup.py:12]: should bump the version be in different patch?

**Comment by dougsland** [comment]
In general LGTM, just a question raised.

File: src/bindings/python/setup.py (hunk 1 above)
Comment (engelmi) [reply src/bindings/python/setup.py:12]: The `setup.py` is generated based on the `setup.py.in` when building the python package - so it doesn't really matter (automated the building and publishing on release). Maybe its even better to remove it?!

Comment (mwperina) [reply src/bindings/python/setup.py:12]: Better to remove this change, changing a version in a non-release commit is confusing

Comment (engelmi) [reply src/bindings/python/setup.py:12]: ~Removed the auto-generated `setup.py` and added it to .gitignore so it won't be added accidentally later.~
Had to revert that, since the rpm build requires the generated file as part of a commit. The version update to v0.7.0 should've been done during the release, but apparently the job checking this didn't work - have to look at that. Lets keep the change.

**Review by mwperina — APPROVED** [review]
LGTM
